src/crs.ts

```diff
@@ -35,7 +35,13 @@ export const CRS_EPSG4326: CRS = {
   opengisUrl: 'http://www.opengis.net/def/crs/EPSG/0/4326',
 };
 
-export const CRS_WGS84: CRS = CRS_EPSG4326;
+export const CRS_WGS84: CRS = {
```

Comment:
Should we add a comment that this is the same as WGS84 / EPSG:4326, only with flipped coordinates?

Reply:
Sure.

Reply:
Thanks. Resolving